bruno/Core/Core Stability Check.bru

```diff
@@ -5,7 +5,7 @@ meta {
 }
 
 get {
-  url: https://{{API_URL}}/core/stable-check//:api_version
+  url: {{API_URL}}/core/stable-check//:api_version
```

Comment:
Do we need the double slash here?

Reply:
No that must be a typo. Let me see if I can adjust.